src-v2/Judge/NTS.Judge.Blazor/Pages/Dashboard/Startlist/StartlistTimer.razor

```diff
@@ -8,11 +9,11 @@
 
 @code {
     [Parameter]
-    public string StartTime { get; set; } = default!;
+    public TimeSpan StartTime { get; set; } = default!;
```

Comment:
Why would `StartTime` be a `TimeSpan`? StartTime should be a point in time, when the participant should start. `TimeSpan` represents an interval of time. It makes no sense to use it here, because now you need to know what is this interval relative to.